src/apple/templates.ts

```diff
@@ -20,6 +20,11 @@ export function getSwiftSnippet(dsn: string): string {
   return `        SentrySDK.start { options in
             options.dsn = "${dsn}"
             options.debug = true // Enabled debug when first installing is always helpful
+
+            // Adds IP for users.
+            // For more information, visit: https://docs.sentry.io/platforms/apple/data-management/data-collected/
```

Comment:
Note: I also opened a [docs PR](https://github.com/getsentry/sentry-docs/pull/13579) to tell users that the data collected varies depending on platform